### PR TITLE
Go generate run & fix contract compilation 

### PR DESCRIPTION
**Pull request by RensR**
## Motivation

Go generate wasn't run after some of the latest changes, resulting in pending generation work. One of those changes also moved the go wrapper generation logic, resulting in a broken reference to the Solidity compilation script. 

## Solution

Fixed the reference issue and then ran `go generate ./...`

### CODE REVIEW

File: core/chains/evm/log/mocks/orm.go [hunk 1]
```diff
@@ -15,20 +15,20 @@ type ORM struct {
 	mock.Mock
 }
 
-// CreateBroadcast provides a mock function with given fields: blockHash, blockNumber, txIndex, logIndex, jobID, qopts
-func (_m *ORM) CreateBroadcast(blockHash common.Hash, blockNumber uint64, txIndex uint, logIndex uint, jobID int32, qopts ...pg.QOpt) error {
```
Comment (RensR) [inline core/chains/evm/log/mocks/orm.go:19]: As no mocks of the orm are used this change is not breaking

Comment (jmank88) [reply core/chains/evm/log/mocks/orm.go:19]: Why are we generating a mock that we don't use? IMHO it should be removed.

File: services/vrf/mocks/aggregator_v3_interface.go [hunk 2]
```diff
@@ -0,0 +1,158 @@
+// Code generated by mockery v2.14.0. DO NOT EDIT.
```
Comment (jmank88) [inline services/vrf/mocks/aggregator_v3_interface.go:1]: If we were building, running, and passing tests without these mocks, why add them back?

File: core/mocks/flags.go [hunk 3]
```diff
@@ -0,0 +1,1060 @@
+// Code generated by mockery v2.14.0. DO NOT EDIT.
```
Comment (connorwstein) [inline core/mocks/flags.go:1]: oh wait these are already generated in core/internal/mocks? Via the go_generate I put in there

**Review by connorwstein — CHANGES_REQUESTED** [review]

File: core/chains/evm/log/mocks/orm.go (hunk 1 above)
Comment (RensR) [reply core/chains/evm/log/mocks/orm.go:19]: I'm completely fine with that, I'm just the messenger that ran `go generate` :) I feel like we might have some other mocks that can be removed besides these ones. Worth a ticket to check them all?

File: core/gethwrappers/generation/compile_contracts.sh [hunk 4]
```diff
@@ -3,7 +3,7 @@
 echo "compiling contracts"
 
 CDIR="$(dirname "$0")"
-COMPILE_COMMAND="$CDIR/../../contracts/scripts/native_solc_compile_all"
+COMPILE_COMMAND="$CDIR/../../../contracts/scripts/native_solc_compile_all"
```
Comment (connorwstein) [inline core/gethwrappers/generation/compile_contracts.sh:6]: interesting, not sure how it was working before 👀

File: core/mocks/flags.go (hunk 3 above)
Comment (connorwstein) [reply core/mocks/flags.go:1]: ah I see what happened - you should remove the mockery calls from gethwrappers/go_generate.go, rm -rf core/mocks/ then and then all will be well

File: core/gethwrappers/generation/compile_contracts.sh (hunk 4 above)
Comment (connorwstein) [reply core/gethwrappers/generation/compile_contracts.sh:6]: think it just silently failed before was the issue

Comment (RensR) [reply core/gethwrappers/generation/compile_contracts.sh:6]: Yeah it only prints an error in the console but happily goes through with generation afterwards

File: services/vrf/mocks/aggregator_v3_interface.go (hunk 2 above)
Comment (RensR) [reply services/vrf/mocks/aggregator_v3_interface.go:1]: Removed these with Connors suggestion below

**Review by connorwstein — DISMISSED** [review]

**Review by connorwstein — APPROVED** [review]

**Review by jmank88 — APPROVED** [review]

File: core/chains/evm/log/mocks/orm.go (hunk 1 above)
Comment (jmank88) [reply core/chains/evm/log/mocks/orm.go]: Yeah, if you think there are some more out there, go for it.

**Pull request merged**